src/components/layout/layout.js

```diff
@@ -96,25 +112,9 @@ const Layout = ({
     const [is_livechat_interactive, setLiveChatInteractive] = React.useState(false)
 
     const is_static = type === 'static'
-
-    const Main = styled.main`
-        margin-top: ${(props) =>
-            !type && is_eu_country
-                ? (props.margin_top && `${props.margin_top + cfd_warning_height_desktop}rem`) ||
-                  7 + cfd_warning_height_desktop + `rem`
-                : (props.margin_top && `${props.margin_top}rem`) || `7rem`};
-        background: var(--color-white);
-        height: 100%;
-        position: relative;
-
-        @media ${device.tabletS} {
-            margin-top: ${(props) =>
-                !type && is_eu_country
-                    ? (props.margin_top && `${props.margin_top + cfd_warning_height_tablet}rem`) ||
-                      7 + cfd_warning_height_tablet + `rem`
-                    : (props.margin_top && `${props.margin_top}rem`) || `7rem`};
-        }
-    `
+    //this pages do not show a banner (cfd) on top of nav. so no need to adjust top margin of main for these pages
+    const no_warning_pages = ['static', 'careers']
+    const should_main_use_eu_margin = !no_warning_pages.includes(type) && is_eu_country
```

Comment:
Nice touch. Can keep it neater this way 👍  Though, I think `should_use_eu_margin` should suffice 😅

Reply:
cool. done.